### DISCUSSION TITLE
`@nrwl/nx-cloud` not included in the nrwl monorepo preset

### DISCUSSION THREAD

**Discussion by simon-abbott** [renovatebot/renovate#18622] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

The `nrwl` monorepo preset is configured using a url prefix (see below), but the [`@nrwl/nx-cloud`](https://www.npmjs.com/package/@nrwl/nx-cloud) package (and possibly others, I haven't done a full audit) doesn't specify a source url. This leads to us now having two PRs up: one including all the main nx/nrwl packages (`nx`, `@nrwl/cli`, etc.), and a second one just for `@nrwl/nx-cloud`.

https://github.com/renovatebot/renovate/blob/78c5d2f39add2b44bcec85d450deb00d170d937f/lib/config/presets/internal/monorepo.ts#L159

Ideally the nrwl monorepo preset would match any package that has the url prefix OR starts with `@nrwl/`, but due to the way the monorepo presets are generated I don't know how best to achieve that.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
You could add this URl manually here: https://github.com/renovatebot/renovate/blob/78c5d2f39add2b44bcec85d450deb00d170d937f/lib/modules/datasource/metadata-manual.ts#L99 but the best thing you can do is submit a PR fixing the upstream repo itself

**Reply by simon-abbott** [reply]
> You could add this URl manually here:

Ah, that will work! I'll submit a PR for that shortly.

> but the best thing you can do is submit a PR fixing the upstream repo itself

Unfortunately the `@nrwl/nx-cloud` package is closed source, so I can't submit any PRs, nor do I expect they will add a source url themselves. 🙁

**Reply by simon-abbott** [reply]
Upon further conversation with the Nrwl/nx team they said that "Although @nrwl/nx-cloud typically follows the same major version, the releases aren't actually coordinated with the rest of the @nrwl packages, so it might not make sense to lump these together", so this seems to be a non-issue after all.

**Comment by rarkins** [comment]
Converted to discussion - not a bug
